src/EnergyPlus/OutputReportPredefined.hh

```diff
@@ -1226,913 +1226,7 @@ struct OutputReportPredefinedData : BaseGlobalStruct
-        this->pdchHumidexOccuHourStroke = 0;
-        this->pdstHeatingSETHours = 0;
-        this->pdchHeatingSETHours = 0;
-        this->pdchHeatingSETOccuHours = 0;
-        this->pdchHeatingSETUnmetDuration = 0;
-        this->pdchHeatingSETUnmetTime = 0;
-        this->pdstCoolingSETHours = 0;
-        this->pdchCoolingSETHours = 0;
-        this->pdchCoolingSETOccuHours = 0;
-        this->pdchCoolingSETUnmetDuration = 0;
-        this->pdchCoolingSETUnmetTime = 0;
-        this->pdrCO2Resilience = 0;
-        this->pdstCO2Hours = 0;
-        this->pdchCO2HourSafe = 0;
-        this->pdchCO2HourCaution = 0;
-        this->pdchCO2HourHazard = 0;
-        this->pdstCO2OccuHours = 0;
-        this->pdchCO2OccuHourSafe = 0;
-        this->pdchCO2OccuHourCaution = 0;
-        this->pdchCO2OccuHourHazard = 0;
-        this->pdrVisualResilience = 0;
-        this->pdstIllumHours = 0;
-        this->pdchIllumHourDark = 0;
-        this->pdchIllumHourDim = 0;
-        this->pdchIllumHourAdequate = 0;
-        this->pdchIllumHourBright = 0;
-        this->pdstIllumOccuHours = 0;
-        this->pdchIllumOccuHourDark = 0;
-        this->pdchIllumOccuHourDim = 0;
-        this->pdchIllumOccuHourAdequate = 0;
-        this->pdchIllumOccuHourBright = 0;
-        this->sizeReportName = 0;
-        this->numReportName = 0;
-        this->sizeSubTable = 0;
-        this->numSubTable = 0;
-        this->sizeColumnTag = 0;
-        this->numColumnTag = 0;
-        this->sizeTableEntry = 0;
-        this->numTableEntry = 0;
-        this->sizeCompSizeTableEntry = 0;
-        this->numCompSizeTableEntry = 0;
-        this->sizeShadowRelate = 0;
-        this->numShadowRelate = 0;
-        this->TotalNotMetHeatingOccupiedForABUPS = 0.0;
-        this->TotalNotMetCoolingOccupiedForABUPS = 0.0;
-        this->TotalNotMetOccupiedForABUPS = 0.0;
-        this->TotalTimeNotSimpleASH55EitherForABUPS = 0.0;
-        this->TotalVozMax = 0.0;
-        this->TotalAnyZoneBelowVozDynForOA = 0.0;
-        this->TotalAllZonesAtVozDynForOA = 0.0;
-        this->TotalAnyZoneAboveVozDynForOA = 0.0;
-        this->TotalAnyZoneBelowVozDynOccForOA = 0.0;
-        this->TotalAllZonesAtVozDynOccForOA = 0.0;
-        this->TotalAnyZoneAboveVozDynOccForOA = 0.0;
-        this->TotalAnyZoneVentUnoccForOA = 0.0;
-        this->TotalAnyAirLoopBelowVozDynForOA = 0.0;
-        this->TotalAllAirLoopsAtVozDynForOA = 0.0;
-        this->TotalAnyAirLoopAboveVozDynForOA = 0.0;
-        this->TotalAnyAirLoopBelowVozDynOccForOA = 0.0;
-        this->TotalAllAirLoopsAtVozDynOccForOA = 0.0;
-        this->TotalAnyAirLoopAboveVozDynOccForOA = 0.0;
-        this->TotalAnyAirLoopVentUnoccForOA = 0.0;
-        this->reportName.deallocate();
-        this->subTable.deallocate();
-        this->columnTag.deallocate();
-        this->tableEntry.deallocate();
-        this->CompSizeTableEntry.deallocate();
-        this->ShadowRelate.deallocate();
+        *this = OutputReportPredefinedData();
```

Comment:
I thought I would need to add `pdchIntFenParent` as a new variable, but it was already there. Trimmed down the `clear_state` function anyway.